pages/admin/editUser.ts

```diff
@@ -1,6 +1,24 @@
 import { Page, expect } from '@playwright/test'
 import { BasePage } from '../basePage'
 
+export const qualifactions = ['PIPE', 'Emergency APs', 'Limited access offenders', 'ESAP']
```

Comment:
Think there's a typo here

```suggestion
export const qualifications = ['PIPE', 'Emergency APs', 'Limited access offenders', 'ESAP']
```

Reply:
good spot!